lib/plugins/package/lib/zipService.js

```diff
@@ -19,22 +25,26 @@ module.exports = {
     output.on('open', () => {
       zip.pipe(output);
 
-      this.serverless.utils.walkDirSync(servicePath).forEach((filePath) => {
```

Comment:
Happy to see this line gone, I timed it and it was taking about 400ms just to test every file in `node_modules`. Hopefully globe isn't as slow.